app/routes/speak.tsx

```diff
@@ -35,9 +35,14 @@ export default function About() {
 			<UnorderedList>
 				<li>
 					Send us your slides 24 hours in advance, so we can check them for
-					accessibility
+					accessibility and/or dry run your talk with us the week before
 				</li>
+				<li>Arrive at the meetup before it starts</li>
 				<li>Be prepared to chat about your talk with attendees</li>
+				<li>
+					Optional: Provide us with an avatar, name, description, and URL
+					(social, website, etc.)
```

Comment:
We've been treating the first few as required for social media & Eventbrite, I think?

```suggestion
					Provide us with a name, talk description, talk title, and optionally: avatar, personal URL, and any social media profiles to share
```

Reply:
Done!